packages/api/src/command/medical/patient/consolidated-webhook.ts

```diff
@@ -65,6 +65,7 @@ export const processConsolidatedDataWebhook = async ({
     await processRequest(
```

Comment:
added cxId to processRequest to follow specs Raf laid out in his issue comment. Required updating devices function calls of processRequest to include cxId too

Reply:
I don't remember why I added it there, and can't think of a reason to have the cxId there